src/ccontrol/UserQuerySelect.cc

```diff
@@ -604,7 +607,7 @@ void UserQuerySelect::qMetaRegister(std::string const& resultLocation, std::stri
         if (not _qSession->containsTable(itr->first, itr->second)) {
             // table either does not exist or it is being deleted, we must stop
             // here but we must mark query as failed
-            _qMetaUpdateStatus(qmeta::QInfo::FAILED, 0, 0);
+            _qMetaUpdateStatus(qmeta::QInfo::FAILED, 0, 0, 0);
```

Comment:
You may avoid explicitly passing 3 zeroes here if you give the default values `=0` to the corresponding parameters of the method `_qMetaUpdateStatus`.